### PR TITLE
Remove dup model info in sms router.conf

### PR DESCRIPTION
**Pull request by fambon**
Device «model» information (`<man-id>:<mod-id>`) is duplicated in files `conf/sms_router.conf` and `conf/device.conf`.
- remove model info from source file `sms router.conf`
- generate line `model <man-id>:<mod-id>` at install time instead

### CODE REVIEW

**Comment by UBiqube-ydu** [comment]
This is a good idea Olivier !
It makes me think that we can have one description file for generating both two files device.properties and sms_router.conf in order to insure consistency.

**Review by ofa- — APPROVED** [review]

**Review by abr-ubiqube — APPROVED** [review]

File: bin/da_installer [hunk 1]
```diff
@@ -197,19 +195,13 @@ read_properties_files() {
 	MAN_NAME=$(read_property "$device_properties" "manufacturer.name")
 }
 
-check_sms_router_conf() {
+patch_sms_router_conf() {
```
[the remark this answers is not in the document]
Comment (fambon) [reply bin/da_installer:198]: ok, thanks for the heads-up !

@UBiqube-ydu can you confirm SMS no longer expects to find man_id / mod_id in sms_router.conf ?

**Review by n1654 — APPROVED** [review]

File: bin/da_installer (hunk 1 above)
[the remark this answers is not in the document]
Comment (UBiqube-ydu) [reply bin/da_installer:198]: @fambon SMS still needs man_id / mod_id in sms_router.conf.

[the remark this answers is not in the document]
Comment (fambon) [reply bin/da_installer:198]: ok, so leaving `patch_sms_router_conf()` in `da_installer`

**Comment by ofa-** [comment]
rebased on top of current master. @asi-ubi good for you ?

**Comment by fambon** [comment]
Suggested improvement not considered interesting in 6 month.
Closing PR

**Pull request closed without merging**